algebraic-graphs.cabal

```diff
@@ -155,7 +155,7 @@ test-suite test-alga
     if !impl(ghc >= 8.0)
         build-depends:  semigroups   >= 0.18.2  && < 0.18.4
     if impl(ghc >= 8.0.2)
-        build-depends:  inspection-testing >= 0.4 && < 0.4.2
+        build-depends:  inspection-testing >= 0.4.2.2 && < 0.5
```

Comment:
`0.4.2.2` has a `base` bound wide enough to allow projects using `algebraic-graphs` to build with GHC 8.8.